src/regtech_data_validator/phase_validations.py

```diff
@@ -919,14 +933,13 @@ def get_phase_1_and_2_validations_for_lei(context: dict[str, str] | None = None)
                     has_no_conditional_field_conflict,
                     id="E2016",
                     name="pricing_init_rate_period.conditional_field_conflict",
-                    description=(
-                        "When 'interest rate type' does not equal 3 (initial rate "
-                        "period > 12 months, adjustable interest), 4 (initial rate "
-                        "period > 12 months, fixed interest), 5 (initial rate period "
-                        "<= 12 months, adjustable interest), or 6 (initial rate period "
-                        "<= 12 months, fixed interest), 'initial rate period' must "
-                        "be blank. When 'interest rate type' equals 3, 4, 5, or 6, "
-                        "'initial rate period' must not be blank."
+                    description=dedent(
+                        """\
+                        * When 'interest rate type' does **not** equal 3 (initial rate period > 12 months, adjustable interest), \
+                        4 (initial rate period > 12 months, fixed interest), 5 (initial rate period <= 12 months, adjustable interest), \
```

Comment:
It _should_ be valid Markdown without the trailing `\`. Is that needed to keep in sync with the CSV?

Reply:
The \ are there to break the string up in the python file so you don't get a single line that looks like "* When 'interest rate type' does **not** equal 3 (initial rate period > 12 months, adjustable interest), 4 (initial rate period > 12 months, fixed interest), 5 (initial rate period <= 12 months, adjustable interest)..."  Which technically should break our linting but black has issues with long string reformatting.